book/src/developers/quick_setup.md

```diff
@@ -59,7 +61,61 @@ View CLI options:
 cargo run -- --help
 ```
 
-### Run locally
+## Building on Arch Based Systems
+
+Before starting, update your system.
+
+```bash
+sudo pacman -Syu
+```
+
+Then, install rust.
+
+```bash
+sudo pacman -S rustup
+rustup install stable
+```
+
+To check that the rust toolchain was successfully installed, run:
+
+```bash
+rustc --version
+```
+
+You should see something like:
+
+```bash
+rustc 1.71.1 (eb26296b5 2023-08-03)
+```
+
+Next, install the required dependencies:
+
+```bash
+sudo pacman -S openssl clang pkg-config base-devel llvm git
+```
+
+Now you can build, test and run Trin!
+
+```bash
+git clone https://github.com/ethereum/trin.git
+cd trin
+
+# Build
+cargo build --workspace
+
+# Run test suite
+cargo test --workspace
+
+# Build and run test suite for an individual crate
+cargo build -p trin-history
+cargo test -p trin-history
+
+# Run
```

Comment:
This headline feels a bit off to me. Technically, to "Run" trin it's just `cargo run`. `cargo run -- --help` is to run the help command. It feels like it's worth at least specifying here, or just including both `# Run help` as `cargo run -- --help` and `# Run Trin with defaults` as `cargo run`